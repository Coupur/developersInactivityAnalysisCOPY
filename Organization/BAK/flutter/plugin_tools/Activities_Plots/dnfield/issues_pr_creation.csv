id,date,creator_login
371742422,2018-10-18 22:46:26,dnfield
371750853,2018-10-18 23:24:47,dnfield
372067921,2018-10-19 18:29:28,dnfield
372088214,2018-10-19 19:32:32,dnfield
