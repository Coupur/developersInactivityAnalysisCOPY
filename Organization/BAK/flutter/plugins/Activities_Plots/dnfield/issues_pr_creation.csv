id,date,creator_login
371169758,2018-10-17 16:45:52,dnfield
376665948,2018-11-02 04:30:03,dnfield
376914442,2018-11-02 18:07:18,dnfield
377028725,2018-11-03 06:36:13,dnfield
382378475,2018-11-19 20:22:09,dnfield
400960225,2019-01-19 04:38:31,dnfield
