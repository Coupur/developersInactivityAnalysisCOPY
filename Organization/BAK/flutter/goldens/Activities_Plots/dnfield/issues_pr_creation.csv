id,date,creator_login
436904280,2019-04-24 20:59:06,dnfield
